src/app/space/add-app-overlay/add-app-overlay.component.spec.ts

```diff
@@ -327,6 +327,35 @@ describe('AddAppOverlayComponent', () => {
       });
     });
 
+    it('continue button should be disabled on navigation in progress', async(() => {
+      const element: HTMLElement = fixture.debugElement.nativeElement;
+      let btnElem = element.querySelector('.code-imports--step_toolbar > button');
```

Comment:
Can the whole "UI" interaction be encapsulated in one component so that when it changes we have only one place to maintain? Or at least make a function which can be re-used in those two tests.